tests/test_settings.py

```diff
@@ -160,7 +161,7 @@
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
 )
 
-LOGIN_URL = reverse_lazy('account_login')
+LOGIN_URL = reverse_lazy('login')
```

Comment:
Not sure why this `LOGIN_URL` value was incorrect, but it was preventing tests from running in my local environment. `'login'` is the correct URL name.